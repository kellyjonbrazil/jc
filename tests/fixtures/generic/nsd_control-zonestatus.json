[{"zone":"sunet.se","status":{"state":"ok","served-serial":"2023091302 since 2023-09-14T00:50:11","commit-serial":"2023091302 since 2023-09-14T07:04:05","wait":"27023 sec between attempts"}},{"zone":"sunet.dev","status":{"state":"refreshing","served-serial":"2023095893 since 2024-10-10T10:10:35","commit-serial":"2023095893 since 2024-10-18T10:00:45","notified-serial":"2023095880 since 2024-10-19T06:11:35","transfer":"TCP connected to 192.168.1.100"}},{"zone":"catz.sunet.se","status":{"pattern":"example.catalog","catalog-member-id":"4b6f6ce2de5929e4.zones.example.catalog.","state":"ok","served-serial":"1705484863 since 2024-01-17T13:06:02","commit-serial":"1705484863 since 2024-01-17T13:06:02","wait":"21341 sec between attempts"}}]
