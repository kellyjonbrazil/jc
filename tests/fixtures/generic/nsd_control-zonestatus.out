zone:	sunet.se
	state: ok
	served-serial: "2023091302 since 2023-09-14T00:50:11"
	commit-serial: "2023091302 since 2023-09-14T07:04:05"
	wait: "27023 sec between attempts"
